templates/bot/js/command-and-response/README.md

```diff
@@ -1,35 +1,35 @@
-# Responding to chat commands
+# Overview of the Command bot template
```

Comment:
Capitalize the first letter?

Reply:
Was following the pattern from workflow bot